CONTRIBUTING.md

```diff
@@ -1,4 +1,18 @@
-# Contribution guidelines for the Streams Standard
+# Streams Standard contributor guidelines
```

Comment:
@domenic "Streams standard"?

Reply:
Nah, in this case "Streams Standard" is a proper noun.